Jenkinsfile

```diff
@@ -5,5 +5,8 @@ cordaPipeline(
     publishOSGiImage: true,
     dailyBuildCron: 'H 03 * * *',
     publishRepoPrefix: 'engineering-tools-maven',
-    publishToMavenS3Repository: true
+    publishToMavenS3Repository: true,
+    javaVersion: '17',
+    workerBaseImageTag: '17.0.4.1-17.36.17',
+    snykDelta: false
```

Comment:
You can drop these last 2 values as we want snykDelta active which is the default.
and workerBaseImageTag has since been removed from the underlying framework so that will now do nothing, it defers to Gradle logic for the source of truth (which you have already updated)